M2/Macaulay2/packages/AlgebraicSplines.m2

```diff
@@ -0,0 +1,2747 @@
+-- Copyright 2014-2015: Mike Dipasquale
+-- You may redistribute this file under the terms of the GNU General Public
+-- License as published by the Free Software Foundation, either version 2
+-- of the License, or any later version.
+
+------------------------------------------
+------------------------------------------
+-- Header
+------------------------------------------
+------------------------------------------
+
+if version#"VERSION" <= "1.4" then (
+    needsPackage "FourierMotzkin"
+    )
+
+newPackage select((
+    "AlgebraicSplines",
+        Version => "0.1.0", 
+        Date => "27. May 2015",
+        Authors => {
+            {Name => "Mike DiPasquale", Email => "midipasq@gmail.com", HomePage => "http://illinois.edu/~dipasqu1"},
+            {Name => "Gwyn Whieldon", Email => "whieldon@hood.edu", HomePage => "http://cs.hood.edu/~whieldon"},
+	    {Name => "Eliana Duarte", Email => "emduart2@illinois.edu", HomePage => "http://illinois.edu/~emduart2"},
+	    {Name => "Daniel Irving Bernstein", Email=> "dibernst@ncsu.edu", HomePage =>"http://www4.ncsu.edu/~dibernst"}
+        },
+        Headline => "Package for computing topological boundary maps and piecewise continuous splines on polyhedral complexes.",
```

Comment:
The prefix "Package for computing" seems redundant, since all packages are packages to compute something.  Also, the headline would look better without the terminal period.